src/Turtle/Prelude.hs

```diff
@@ -575,6 +588,44 @@ touch file = do
 #endif
         else output file empty )
 
+type Permissions = Filesystem.Permissions -> Filesystem.Permissions
```

Comment:
I would just skip the type synonym.  The fully expanded type is not that bad and it will avoid confusion